chain/cosmos/genesis.go

```diff
@@ -0,0 +1,32 @@
+package cosmos
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/icza/dyno"
+	"github.com/strangelove-ventures/interchaintest/v7/ibc"
+)
+
+func ModifyGenesisProposalTime(votingPeriod string, maxDepositPeriod string) func(ibc.ChainConfig, []byte) ([]byte, error) {
```

Comment:
This works for gov modules that are on v1beta1.
Gov v1 will look like this, so maybe break out into two functions with a Gov v1beta1 vs v1 name?
```
dyno.Set(g, votingPeriod, "app_state", "gov", "params", "voting_period")
dyno.Set(g, maxDepositPeriod, "app_state", "gov", "params", "max_deposit_period")
dyno.Set(g, chainConfig.Denom, "app_state", "gov", "params", "min_deposit", 0, "denom")
```

Reply:
Ahh. Good eye!
Will take a pass at this.

Reply:
@misko9
Ok, added gov module version check.